message_converter/src/converter_node.cpp

```diff
@@ -244,8 +234,16 @@ int main(int argc, char **argv)
     // Values in comments are from the parking lot test
     // double home_latitude = 42.3711071377; 
     // double home_longitude = -71.2168400352; 
-    // double home_altitude = -15.3775837652; 
+    // double home_altitude = -15.3775837652;
+    // double home_latitude = 40.8502544178; 
+    // double home_longitude = 14.3080716592; 
+    // double home_altitude = 77.7699139992;     
     SwiftNavRover obj(&n, home_latitude, home_longitude, home_altitude);
-    ros::spin();
+    ros::Rate loop_rate(10);
```

Comment:
i'd pump this up to at least 20 or 25; we want to service it faster than we're getting measurements

Reply:
Addressed in https://github.com/Humatics/ethz_piksi_ros/pull/1/commits/c275451451b84a2384550d93f7675d8dbdde6f24